posthog/management/commands/run_autoreload_celery.py

```diff
@@ -0,0 +1,39 @@
+import django
+
+from django.core.management.base import BaseCommand
+
+from posthog.tasks.utils import CeleryQueue
+
+# ❗ needs to be called *before* importing autoreload
+django.setup()
+
+from django.utils import autoreload  # noqa: E402
+
+
+class Command(BaseCommand):
+    help = "Wrap celery in djangos auto reload functionality"
+
+    def handle(self, *args, **options):
+        autoreload.run_with_reloader(self.run_celery_worker)
+
+    def run_celery_worker(self):
+        from posthog.celery import app as celery_app
+
+        queues = [q.value for q in CeleryQueue]
+
+        args = [
+            "-A",
+            "posthog",
+            "worker",
+            "-B",
+            "-S",
+            "redbeat.RedBeatScheduler",
```

Comment:
The beat command didn't work for me.

I then saw it's possible to just run the beat within the worker for development purposes. This should be alright for us?